whittle/models/gpt/blocks/mlp.py

```diff
@@ -35,6 +38,7 @@ def set_sub_network(
         )
 
     def reset_super_network(self):
+        """Reset the input dimensionality of the current sub-network to the super-network dimensionality."""
```

Comment:
same as above (maybe I'm wrong tho, feel free to correct)

Reply:
This one mentions the current sub-network, it is already correct isn't it?

Reply:
When I was referring to the above comment I meant the `input` being out of place :D This function does not just reset the input size but also the intermediate so describing it as just resetting the input size would be incorrect, right?

Reply:
I agree